docker-compose.yml

```diff
@@ -28,6 +37,8 @@ services:
 
   web:
     build: .
+    expose:
+    - "5000"
```

Comment:
@aviaryan In the above web directory, the application is fired up on port 5000 and then the same port is then forwarded to port 80 on the host environment, but for this service we should add environment variables to persist our container running from the resulting image.